.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: java
+
+dist: trusty
+
+jdk:
+- oraclejdk8
```

Comment:
`openjdk8` makes trouble with javafx stuff because it has been decoupled `openjdk` and `openjfx`.
so javafx project must be built using `oraclejdk8` for preventing openjfx symbol/syntax errors. because `oraclejdk8` contains openjfx stuff.

As I mentioned few days ago, I said `oraclejdk8` makes trouble with license thing.
this is because build environment.

the default `dist` is `xenial` on `TravisCI`. and `xenial` seems using jdk 1.8.212+ (I guess) which is commercial license
but `trusty` dist is using jdk 1.8.151 which is non-commercial licensed.
so I specified it as `trusty`.

about more `dist` things, you can check out [here](https://docs.travis-ci.com/user/languages/java)